src/clj/rems/handler.clj

```diff
@@ -67,6 +67,13 @@
       (apply-for-resources resource)
       (apply-for-resources [resource])))
 
+  (GET "/landing_page" [] ; DEPRECATED: legacy url redirect
```

Comment:
Perhaps we could put that to the release notes as deprecated too and then remove it, since we dropped the legacy auth support too?

Reply:
yeah I guess we could just drop it, the any links to /landing_page should be long gone now